package-lock.json

```diff
@@ -1,6120 +1,8 @@
 {
   "name": "@bigcommerce/api-nodejs",
   "version": "0.1.1-alpha",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Oops, looks like you might be using an outdated version of NPM. We want to us NPM 7 or greater - can you please switch NPM versions to something more recent and upgrade your lockfile in this PR from v1 → v2?